### ISSUE TITLE
[Feature request] ssllabs.com "Chain issues. Contains anchors" like support

### ISSUE THREAD

**Issue by nomandera** [drwetter/testssl.sh#1842]
If am not sure if this is in scope but put it here for consideration.

I have just completed a deployment using testssl.sh as usual to spot issues and attempt to achieve the highest score.

In this instance when I conducted the final "one off sanity check" SSL Labs test i noticed it reported a `Chain issues. Contains anchors` warning.

This link opened by someone else is an exact match to the situation

https://qualys-secure.force.com/discussions/s/question/0D52L00004TnvJ5SAJ/chain-issues-contains-anchors

The fix was as you might expect to identify the certificate and simply delete it from the Apache chain.

**Comment by drwetter** [comment]
yes, it would be great to say so. It IS identified already if you look carefully (found this one at SSLlabs):

![image](https://user-images.githubusercontent.com/8036727/107856934-ce605180-6e2b-11eb-8801-3f45c8b59c48.png)
